### DISCUSSION TITLE
Accidentally deleted dat apps

### DISCUSSION THREAD

**Discussion by NadineNabih** [dat-linux/community#76] [Issues]
let tell you what happeded from beginning , when i did install dat i didnt like lxqt so i tried downloading gnome , it didnt work well and tried kde , i liked it and continued using it until it is perfect , i had storage issues i tried finding ways to remove lxqt and gnome , lxqt worked well , i tried removing gnome i used //sudo apt-get remove --purge gnome*
well lets just say that it did remove all apps related to dat and kept gnome somehow
is there a fix or am i cooked

**Comment by dat-linux** [comment]
Try this:

`sudo apt install --reinstall datlinux-ctrl-pnl`

Control panel can be launched with:

`/usr/local/bin/datlinux/dat-dsapps.py`

**Reply by NadineNabih** [reply]
i will try and update you

**Reply by NadineNabih** [reply]
update : it did download 😍

thanks

**Reply by ramblas98** [reply]
Hello, when I run the application I get the following error message:

`/usr/local/bin/datlinux/dat-dsapps.py

I got the lock
Traceback (most recent call last):
  File "/usr/local/bin/datlinux/dat-dsapps.py", line 1332, in <module>
    loc_vars['cou'] = tr[f"{lang}_cou"]
                      ~~^^^^^^^^^^^^^^^
KeyError: 'es_cou'
`

**Reply by oss-linux-etc** [reply]
A fix is up for this if you update your system

**Reply by ramblas98** [reply]
With the update the panel does work!!!

**Comment by oss-linux-etc** [comment]
Changing desktops shouldn't affect the underlying application. But it can mess with your menu items.

**Reply by NadineNabih** [reply]
and this is true actually the issue occured when deleting gnome , i did not know that this means dat apps not gnome itself

**Reply by mlrx-potindji** [reply]
Please, can you share an image of what your desktop or interface now look like ???
